packages/schemas/src/foundations/jsonb-types.ts

```diff
@@ -72,6 +73,22 @@ export const userLogPayloadGuard = z.object({
 
 export type UserLogPayload = z.infer<typeof userLogPayloadGuard>;
 
+/**
+ * Logs
+ */
+
+export const logPayloadGuard = z.object({
+  ip: z.string().optional(),
+  userAgent: z.string().optional(),
+  applicationId: z.string().optional(),
+  connectorId: z.string().optional(),
+  userId: z.string().optional(),
+  request: z.object({ body: z.object({}).optional() }).optional(),
+  response: z.object({ body: z.object({}).optional() }).optional(),
```

Comment:
i also meant `request` and `response` fields

Reply:
See Slack thread [discussion](https://silverhand-io.slack.com/archives/C02CMB8JVJB/p1650016973075049)